azext_edge/tests/settings.py

```diff
@@ -16,6 +16,7 @@ class EnvironmentVariables(Enum):
     instance = "azext_edge_instance"
     context_name = "azext_edge_context_name"
     kv = "azext_edge_kv"
+    user_assigned_mi_id = "azext_edge_user_assigned_mi_id"
```

Comment:
Is this variable passed into / set in a pipeline somewhere, or how is it configured?

Reply:
no... 
it currently makes a new mi... 
do I need to add in a pass through... 
can I not...